hash_to_array.rb

```diff
@@ -0,0 +1,33 @@
+
+
+def collector 
+keys_array = Array.new
+values_array = Array.new
+puts "Enter 5 keys please"
+5.times do
+keys_array.push(gets.chomp)
+end
+puts "Enter 5 values please"
+5.times do
+values_array.push(gets.chomp)
+end
+hash = {}
+iter =0
+5.times do
```

Comment:
you could save one loop here by just setting the hash keys and values in the loop when you collect them